db/fixtures/miq_shortcuts.yml

```diff
@@ -484,3 +484,8 @@
   :url: /alerts_most_recent
   :rbac_feature_name: monitor_alerts_most_recent
   :startup: true
+- :name: miq_proxy_tasks
+  :description: Configure / Tasks
```

Comment:
@GregP couple of minor changes, can we change description to "Settings / Tasks" to match the features tree. Also in the url below it should be ` /miq_task/index?jobs_tab=tasks ' jobs_tab=alltasks is not a valid value.